src/odin/ingestion/qlik/cubic_archive.py

```diff
@@ -90,15 +87,7 @@ def __init__(self, table: str) -> None:
             self.save_local = False
 
         self.export_folder = os.path.join(DATA_SPRINGBOARD, CUBIC_QLIK_DATA, self.table)
-        self.reset_tmpdir(make_new=True)
-
-    def reset_tmpdir(self, make_new: bool = True) -> None:
-        """Reset TemporaryDirectory folder."""
-        if hasattr(self, "_tdir"):
-            self._tdir.cleanup()  # type: ignore
-        if make_new:
-            self._tdir = tempfile.TemporaryDirectory()
-            self.tmpdir = self._tdir.name
```

Comment:
Moved this to OdinJob to make the tempdir functionality available to all OdinJob objects be default.